std_msgs/Header header

# String class IDs for each object
string[] class_names

# depths of each object
float32[] depths

# size of each object
float32[] size
